src/neuroconv/utils/str_utils.py

```diff
@@ -0,0 +1,47 @@
+def human_readable_size(size_bytes: int, binary=True) -> str:
+    """
+    Convert a file size given in bytes to a human-readable format.
+
+    Parameters
+    ----------
+    size_bytes : int
+        The size in bytes.
+    binary : bool, optional
+        If True, use binary prefixes (KiB, MiB, etc.). If False, use SI prefixes
+
+    Returns
+    -------
+    str
+        A human-readable string representation of the size.
+
+    Examples
+    --------
+    >>> human_readable_size(123)
+    '123 B'
+    >>> human_readable_size(1234)
+    '1.21 KB'
+    >>> human_readable_size(123456789)
+    '117.74 MB'
```

Comment:
With default of `binary=True` these would be `iB` - left the last one to demonstrate that usage

```suggestion
    >>> human_readable_size(123)
    '123 B'
    >>> human_readable_size(1234)
    '1.21 KiB'
    >>> human_readable_size(123456789, binary=False)
    '123.46 MB'
```